integrations/access/opsgenie/client.go

```diff
@@ -160,7 +160,7 @@ func (og Client) CreateAlert(ctx context.Context, reqID string, reqData RequestD
 
 func (og Client) getResponders(reqData RequestData) []Responder {
 	schedules := og.DefaultSchedules
-	if reqSchedules, ok := reqData.SystemAnnotations[types.TeleportNamespace+types.ReqAnnotationSchedulesLabel]; ok {
+	if reqSchedules, ok := reqData.SystemAnnotations[types.TeleportNamespace+types.ReqAnnotationNotifyServicesLabel]; ok {
```

Comment:
Have made the variables in the opsgenie plugin more consistent. The `defaultSchedules` that belongs to the opsgenie client is used in /e as well so to change it we might need to make some extra field fill both in then remove once /e is updated. 
The `Responders` field is called responder instead of schedules because in Opsgenie it can be either a user or a schedule and copied their naming convention to match the field in the API and on the web page for clarity. Would it make more sense to swap over to schedules since that is the only way we use them?